src/transformers/models/mixtral/configuration_mixtral.py

```diff
@@ -93,6 +93,8 @@ class MixtralConfig(PretrainedConfig):
             allow the model to output the auxiliary loss. See [here]() for more details
         router_aux_loss_coef (`float`, *optional*, defaults to 0.001):
             The aux loss factor for the total loss.
+        router_jitter_noise (`float`, *optional*, defaults to 0.01):
```

Comment:
thanks for following the API conventions! Let's default to 0 to make sur this is BC

Reply:
I've defaulted it to `0.0` as it's a float, and to keep it consistent with the other args 🙂